packages/browser/tsconfig.test.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "./tsconfig.json",
+
+  "include": ["test/**/*"],
+
+  "compilerOptions": {
+    // should include all types from `./tsconfig.json` plus types for all test frameworks used
```

Comment:
Would just 🔥 these comments in all of the tsconfigs, we can leave it in the template

Reply:
Unlike the other comments, which I removed, these I think we should keep, because if you ever were to make changes, you'd need to know this fact.